docs/user_manual.md

```diff
@@ -0,0 +1,76 @@
+# User manual
+
+## Contents
+
+- [Installation](#installation)
+  - [Dependencies](#dependencies)
+  - [Windows](#windows)
+
+## Installation
+
+### Dependencies
+
+- Python (3.8, 3.9)
+- [`virtualenv`](https://virtualenv.pypa.io/en/latest/) (optional, but recommended)
+- [Git](https://git-scm.com/downloads)
+
```

Comment:
```suggestion
- [Git](https://git-scm.com/downloads)
- [Git LFS](https://git-lfs.github.com)

```
Maybe mention it?

Reply:
Was thinking of it. @R0flcopt3r  did you have lfs on your VM?

Relating I was thinking it would make installation easier, if we had a wheel to point to.

Reply:
We could host them here in the repo.  But seems that things are again limited on space/transfer, etc.  Not sure how much bandwidth we need.  I'm not sure publishing to PyPI is necessary that this is a package that only a few need.  Then it's easier to control where this is hosted/served.

Ref.: https://github.com/features/packages#pricing

Reply:
yeah, my thinking was attached artifact to tags

Reply:
I'll test now and report back the findings.

Reply:
Git should no longer be needed, ref #101.